Resources/datasets/fundamentals/direct-access.html

```diff
@@ -15,11 +15,11 @@
 var nb = QuantConnect.Symbol.Create("NB", SecurityType.Equity, Market.USA);
 var fundamentals = Fundamentals(new List&lt;Symbol&gt;{ nb, ibm }).ToList();</pre>
         <pre class="python"># Single asset
-ibm = Symbol.Create("IBM", SecurityType.Equity, Market.USA)
+ibm = QuantConnect.Symbol.Create("IBM", SecurityType.Equity, Market.USA)
```

Comment:
Python doesn't need `QuantConnect.`, since `Symbol` is not ambiguous as it is in C#.